api/build.gradle

```diff
@@ -484,7 +484,9 @@ dependencies {
   generatedCompile 'io.swagger.core.v3:swagger-annotations:2.0.0'
   generatedCompile 'io.gsonfire:gson-fire:1.8.3'
   generatedCompile 'org.threeten:threetenbp:1.3.5'
+  generatedCompile group: 'javax.xml.bind', name: 'jaxb-api', version: '2.3.1'
```

Comment:
nit: I don't have a preference on the syntax but we've been using the 'group:name:version' syntax for most of our dependencies above